README.md

```diff
@@ -1,2 +1,115 @@
-# ffc-pay-file-consumer
-FFC Pay consume file transfer requests from DAX
+# FFC Pay file consumer
+FFC Pay service to transfer files from Dynamics 365 (DAX) to payment service.
+
+This function is triggered from a service bus message requesting a file transfer from an Azure File Share to Azure Blob Storage.
+
+The message contains the name of the file that should already have been written to a file share location.
+
+If the file is present, the file will be copied and the original will be moved to an archive folder.
+
+## Example message
+
+```
+{ 
+ "AzureAdapterType": "AzureFileStorage", 
+ "AzureFileShare": "fixeddestinationreports", 
+ "AzureSecretName": "storage-account-integration-key", 
+ "AzureStorageAccount": "predaxinfst1202", 
+ "AzureStorageType": "FixedDestinationReports", 
+ "BusinessEventId": "RsfFileToTransferToOnPremBusinessEvent", 
+ "ControlNumber": 5637236081, 
+ "EventId": "708F37F0-D1D9-45FC-BF80-ABE590E5CE28", 
+ "EventTime": "/Date(1643734354000)/", 
+ "MajorVersion": 0, 
+ "MinorVersion": 0, 
+ "OutputFileName": "PFSI_Settlements2022-02-18.csv", 
+ "ProcessingLocation": "REDACTED FOLDER" 
+}
+```
+
+## Prerequisites
+
+- Node.js 16+
+- access to an Azure blob storage account (see options below)
+- access to Azure file share storage account
+- [Azure Functions Core Tools](https://docs.microsoft.com/en-us/azure/azure-functions/functions-run-local?tabs=v4%2Clinux%2Ccsharp%2Cportal%2Cbash)
```

Comment:
Should be give a concrete version number?

Reply:
Done